tests/component.js

```diff
@@ -413,3 +412,85 @@ describe( 'CKEditor Component', () => {
 		} );
 	}
 } );
+
+describe( 'comp on detach elem', () => {
+	let wrapperr;
+
+	afterEach( () => {
+		wrapperr.destroy();
+	} );
+
+	after( () => {
+		return deleteCkeditorScripts();
+	} );
+
+	it( 'tries to mount component on detached element and use default interval strategy before creates', () => {
+		const parent = document.createElement( 'div' );
+		// Vue will replace mount target, so we have extra parent to manipulate it.
+		const mountTarget = document.createElement( 'div' );
+		parent.appendChild( mountTarget );
+
+		wrapperr = createComponent( {}, mountTarget );
+
+		return delay( 100, () => {
+			// Editor is created after namespace loads
+			// so we need to wait for the real results
+			expect( wrapperr.vm.instance ).to.be.null;
+		} ).then( () => {
+			document.body.appendChild( parent );
+		} ).then( () => {
+			return delay( 1000, () => {
+				expect( wrapperr.vm.instance ).to.be.not.null;
+			} );
+		} );
+	} );
+
+	it( 'tries to mount component on detached element and use callback strategy', () => {
+		const parent = document.createElement( 'div' );
+		// Vue will replace mount target, so we have extra parent to manipulate it.
+		const mountTarget = document.createElement( 'div' );
+		parent.appendChild( mountTarget );
+		let createEditor;
+
+		wrapperr = createComponent(
+			{
+				config: {
+					delayIfDetached_callback: finishCreation => {
+						createEditor = finishCreation;
+					}
+				}
+			},
+			mountTarget
+		);
+
+		return delay( 100, () => {
+			// Editor is created after namespace loads
+			// so we need to wait for the real results
+			expect( wrapperr.vm.instance ).to.be.null;
+		} ).then( () => {
+			document.body.appendChild( parent );
+			createEditor();
+		} ).then( () => {
+			return delay( 1000, () => {
+				expect( wrapperr.vm.instance ).to.be.not.null;
+			} );
+		} );
+	} );
+} );
+
+function createComponent( props, mountTarget = document.body ) {
+	const fakeParent = window.document.createElement( 'span' );
```

Comment:
```suggestion
	const fakeParent = document.createElement( 'span' );
```

In all other places, we use `document` alone. I guess it's some leftover from the previous version of tests?

Reply:
Must be... I don't remember to make it on some specific purpose 🤔